core/src/main/scala/com/raphtory/components/partition/QueryExecutor.scala

```diff
@@ -104,6 +103,13 @@ class QueryExecutor(
           graphLens.receiveMessage(msg)
           receivedMessageCount.addAndGet(1)
 
+        case msg: FilteredEdgeMessage                                         =>
+          logger.trace(
+                  s"Job '$jobID' at Partition '$partitionID': Executing 'FilteredEdgeMessage', '$msg'."
+          )
+          graphLens.receiveMessage(msg)
+          receivedMessageCount.addAndGet(1)
```

Comment:
Was the counting moved outside because have the two different types of messages now?

Reply:
Counting was always here for optimising the VertexMessageBatch case?

Reply:
Sorry yes you are correct I was just misremembering - thought I had moved it inside with the prior refactor